src/includes/getting-started-primer/javascript.nextjs.mdx

```diff
@@ -0,0 +1,20 @@
+<Note>
+
+_Sentry's Next.js SDK enables automatic reporting of errors and exceptions._
+
+</Note>
+
+Features:
+
+- Automatic [Error Tracking](/product/issues/) with Source Maps for both JavaScript and TypeScript
+- Events [enriched](/platforms/javascript/enriching-events/context/) with device data
+- [Breadcrumbs](/platforms/javascript/enriching-events/breadcrumbs/) created for outgoing HTTP request with XHR and Fetch, and console logs
+- [Release Health](/product/releases/health/) tracks crash free users and sessions
+- Under the hood the SDK relies on our [React SDK](/platforms/javascript/guides/react/) on the frontend and [Node SDK](/platforms/node) on the backend, which makes all functions available for React and Node also available in this SDK
+
+<Alert level="warning" title="Based on the Next.js plugin system">
+
+The SDK relies on the experimental plugin system of Next.js.
+Changes to the plugin system may prevent the SDK from working as expected.
+
+</Alert>
```

Comment:
Edited to use most standard wording:
```suggestion
<Note>

The SDK relies on the experimental plugin system of Next.js. Changes to the plugin system may prevent the SDK from working as expected. We recognize the irony.

</Note>
```

You can also add a link to a GH Issue or feedback area if useful (as we do [here](https://github.com/getsentry/sentry-cocoa/discussions/1016))